src/common/FileSystem.cpp

```diff
@@ -2353,6 +2353,18 @@ std::string DefaultHomePath()
 #endif
 }
 
+// Determine path to temporary directory
+std::string DefaultTempPath()
```

Comment:
This is wrong as a temp directory. You can't make normal files in this path.

You can see one other use of a temp dir in TestApplication. In that place I got a Windows temp directory with the `tempnam` function which figures out the temp dir on its own, so maybe DefaultTempPath is not a suitable abstraction for Windows anyway.

Reply:
I rewrote the code to not use DefaultTempPath on Windows since it was not using a temp path indeed.

Reply:
Though maybe I would rename `DefaultTempPath` to `DefaultTempPrefix` or something like that because it doesn't return a directory but a prefix for a file name.

Reply:
The thread is no longer relevant to the current implementation.